src/styles/global.css

```
@@ -0,0 +1,121 @@
+}
+
+@layer components {
+  .prose code {
+    &::before,
+    &::after {
+      content: "" !important;
+    }
+  }
+}
+
+@import "./prose.css";
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)